virttest/utils_netperf.py

```diff
@@ -273,6 +307,13 @@ def __init__(self, address, netperf_path, md5sum="", netperf_source="",
                                             password, prompt, linesep, status_test_command,
                                             compile_option, install)
 
+    @property
+    def netserver_bin(self):
+        get_basename = os.path.basename
+        if self.client == "nc":
+            get_basename = ntpath.basename
+        return get_basename(self.netserver_path)
+
```

Comment:
@luckyh, since these lines and L374~L380 are dupped, can we merge them in the parent class `Netperf`?

Reply:
Sorry, they are similar but not the same, so this is good.